src/app/tests/suites/certification/Test_TC_DRLK_2_1.yaml

```diff
@@ -985,8 +1038,11 @@ tests:
       command: "readAttribute"
       attribute: "EnableLocalProgramming"
       response:
-          value: 1
```

Comment:
No, hold on a sec.  Test plan at https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/door_lock_Cluster.adoc#tc-drlk-2-1-attributes-check-dut-server in fact requires this to be 1.  I agree that's broken, but that means we should fix the test plan.  Please check the other changes against it too.

Reply:
Corresponding Test Plan changes are here: https://github.com/CHIP-Specifications/chip-test-plans/pull/3279